docs/starters.yml

```
@@ -90,6 +90,14 @@
     - Includes plugins for easy, beautiful typography
     - Includes React Helmet to allow editing site meta tags
     - Includes plugins for offline support out of the box
+- url: https://github.com/gatsbyjs/gatsby/tree/master/starters/gatsby-starter-blog-theme
```

There's a mirrored repository for this starter: https://github.com/gatsbyjs/gatsby-starter-blog-theme 🙂 

```suggestion
- url: https://github.com/gatsbyjs/gatsby-starter-blog-theme
```